projects/gstreamer/gst-discoverer.c

```diff
@@ -0,0 +1,152 @@
+/*
```

Comment:
So, any chance to have this file upstream?
https://github.com/google/oss-fuzz/blob/master/docs/ideal_integration.md#tldr

Reply:
I was going to setup a new repository for GStreamer for all our CI/QA stuff (still located on freedesktop/gstreamer). I'll migrate the code over there and rebase all previous commits. Thanks for your patience/review :)

Reply:
ok

Reply:
Done, now upstream : https://cgit.freedesktop.org/gstreamer/gst-ci/tree/fuzzing/gst-discoverer.c